src/components/registerForm.js

```diff
@@ -0,0 +1,47 @@
+import { useRef, useState } from "react";
+import { registerService } from "../services/userService";
+import {
+  EmailField,
+  NameField,
+  PasswordField,
+  SubmitButton,
+  ErrorField,
+} from "./elements";
+
+function RegisterForm(props) {
+  const [nameError, setNameError] = useState();
+  const [emailError, setEmailError] = useState();
+  const [passwordError, setPasswordError] = useState();
+  const formRef = useRef();
+  const register = () => {
+    let form = new FormData(formRef.current);
+    registerService(form)
```

Comment:
Same leave a blank line before and after this event handler
```suggestion
    let formData = new FormData(formRef.current);
    registerService(formData)
```